assembl/static2/css/views/administration.scss

```diff
@@ -16,6 +16,10 @@
     position: relative;
   }
 
+  .no-box-shadow {
+    box-shadow: none !important;
```

Comment:
If possible, avoid to use `!important`

Reply:
Doesn't work without it. It's to override the style of the component which we import from react-color